sdk/sqlmanagement/Microsoft.Azure.Management.Sql/src/Generated/Models/Database.cs

```diff
@@ -176,9 +176,12 @@ public Database()
         /// <param name="autoPauseDelay">Time in minutes after which database
         /// is automatically paused. A value of -1 means that automatic pause
         /// is disabled</param>
-        /// <param name="storageAccountType">The storage account type used to
-        /// store backups for this database. Possible values include: 'GRS',
-        /// 'LRS', 'ZRS'</param>
+        /// <param name="currentBackupStorageRedundancy">The storage account
```

Comment:
replacing storageAccountType with the two backupstorageredundancy parameters is a breaking change.